packages/orm/src/relations/ReactiveField.ts

```diff
@@ -95,22 +95,29 @@ export class ReactiveFieldImpl<T extends Entity, H extends ReactiveHint<T>, V>
   }
 
   async load(opts?: { forceReload?: boolean }): Promise<V> {
+    // Even without `forceReload=true`, any explicit calls to `.load()` ==> ensure a fresh value,
+    // because `.get` may have cached the stale/previously-calculated value.
+    //
+    // (Currently `em.populate` / `populateDataLoader` has a hack/escape hatch that ignores calling
+    // RF.load(), to avoid pulling in the RF subgraph over using the materialized value, which ideally would be
+    // replaced with us learning to do the maybeDirty approach.)
```

Comment:
Just this comment since it should apply to the whole `load` method and not just the `if` check